.mergify.yml

```diff
@@ -54,25 +54,26 @@ pull_request_rules:
       - merged
     actions:
       delete_head_branch:
-  - name: automatic squash and merge with success checks and the files matching the regex docker-compose.yml$ are modified.
+  - name: automatic squash and merge with success checks for those automated PRs created by the apmmachine.
     conditions:
+      - -merged
+      - -closed
       - check-success=beats-ci/e2e-testing/pr-merge
       - label=automation
-      - files~=docker-compose\.yml$
+      - author=apmmachine
```

Comment:
For instance:

https://github.com/elastic/e2e-testing/pull/1848 matched the previous behaviour of chjanges in the `docker-compose.yml` but https://github.com/elastic/e2e-testing/pull/1849 somehow is only touching one file


This `author` filtering will allow the merge for the above-mentioned PRs.